functorch/examples/dp_cifar10/cifar10_transforms.py

```diff
@@ -95,18 +95,14 @@ def train(args, model, train_loader, optimizer, epoch, device):
 
         # Step 1: compute per-sample-grads
 
-        # In order to use functional vmap+grad, we need to be able to
-        # pass the weights to a model.
-        func_model, weights = make_functional(model)
-
```

Comment:
Previously there was a way for users to specify that they don't want the parameters to require_grad when returned from `make_functional`, right? This isn't exhibited in the examples, but it should really be used in the dp-cifar10 example to improve performance. 

I'm not really sure what to do about this

Reply:
As it stands the onus is on the user--I'll add this into this example. This should be simple to add to the functorch version of this API (adding on to what you brought up below). Basically this should work if we `.detach()` as we pass these values to vmap or grad, so it should be possible to add a flag to named_parameters or add a helper function (tbh we probably want another helper to avoid the repetition of `{k: v for k, v in model.named_parameters()}`)